config.md

```diff
@@ -28,7 +28,8 @@ Changing it means creating a new derived image, instead of changing the existing
 A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
 Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
 
-NOTE: the DiffID is different than the digest in the manifest list because the manifest digest is taken over the gzipped layer for `application/vnd.oci.image.layer.v1.tar+gzip` types.
+The difference between DiffIDs and the layer digests in the [manifest's `layers`](manifest.md#image-manifest-property-descriptions) is that the layer digest is taken over the blob regardless of compression, while the DiffID is taken after removing any compression.
+For an `application/vnd.oci.image.layer.tar+gzip` layer, the layer digest is taken over the `application/vnd.oci.image.layer.tar+gzip` content, while the DiffID is take over the `application/vnd.oci.image.layer.tar` content.
```

Comment:
> For this PR to make even a remote bit of sense, a layer digest may be taken over `application/vnd.oci.image.layer.tar` or `application/vnd.oci.image.layer.tar+gzip`.

For a generic layer yes, and those types are *still* not exhaustive because we only [SHOULD layer media types][1].  This sentence, on the other hand, is qualified with “For an `application/vnd.oci.image.layer.tar+gzip` layer”, and in that case the layer hash MUST be taken over the `application/vnd.oci.image.layer.tar+gzip` content.

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc3/manifest.md#L60